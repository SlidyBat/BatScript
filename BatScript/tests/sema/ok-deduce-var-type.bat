def f() -> int:
	return 1

def g():
	return 3.14

int_var := 1
print int_var
float_var := 3.14
print float_var
str_var := "Hello"
print str_var
bool_var := true
print bool_var

a := int_var
print a
b := f()
print b
