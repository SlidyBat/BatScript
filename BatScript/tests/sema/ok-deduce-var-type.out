1
3.14
Hello
1
1
1
